src/Discovery/Utils/Search/filterByAdvSearch.ts

```diff
@@ -0,0 +1,63 @@
+import { DiscoveryConfig } from '../../DiscoveryConfig';
+
+interface FilterState {
```

Comment:
this can use the exported interface?

Reply:
Good point! The FilterState here has been updated as an import from the Discovery.tsx file.